app/assets/stylesheets/content/_forms.sass

```diff
@@ -222,8 +222,8 @@ fieldset.form--fieldset
   .form.-vertical &
     @include grid-orient(vertical)
 
-  .grid-block:nth-last-of-type(n+2) > &,
-  .form--row:nth-last-of-type(n+2) > &,
+  .grid-block > &:nth-last-of-type(n+2),
+  .form--row > &:nth-last-of-type(n+2),
   .form--grouping-row > &:nth-last-of-type(n+2)
     padding-right: 1rem
```

Comment:
I find it quite counter-intuitive that a padding is used on something that looks like a margin, but this is probably only me...

It feels like the `nth-last-of-type` is only neccessary, because the `Sign in` button has its own margin on the right, whereas all input fields do not have their own margin. Thus the `form--field`s containing the inputs *add* a padding to compensate for the missing margin of their children (or vice versa: the one containing the button is the only one *missing* the padding).

Just my two (non-CSS-approved) cents...